custom-actions/timeline/Extreme_Reach_IO.py

```diff
@@ -0,0 +1,118 @@
+#Script to set in and out points in a timeline based on the Extreme Reach delivery specs.
+def get_media_panel_custom_ui_actions():
+
+    def scope_clip(selection):
+        import flame
+        for item in selection:
+            if isinstance(item, (flame.PyClip)):
+                return True
+        return False
+
+    def er_io_60(selection):
+        import flame
+        for sequence in selection:
+            i = sequence.duration.frame
+            frameRate = sequence.frame_rate
+            tc_IN = flame.PyTime("00:59:53:00", frameRate)
+            tc_OUT = flame.PyTime("01:01:00:01", frameRate)
+            sequence.in_mark = tc_IN
+            sequence.out_mark = tc_OUT
+
+    def er_io_30(selection):
```

Comment:
There are some code duplication here. You could create a method that take tc_IN and tc_OUT strings and have most of the loop code shared.

Reply:
Could you explain what you mean? I'm still early in my coding career. 😀

Reply:
```suggestion

    def set_selection_in_out_marks(selection, tc_in, tc_out):
        import flame
        for sequence in selection:
            i = sequence.duration.frame
            frameRate = sequence.frame_rate
            sequence.in_mark = flame.PyTime(tc_in, frameRate)
            sequence.out_mark = flame.PyTime(tc_out, frameRate)
            
    def er_io_15(selection):
        set_selection_in_out_marks(selection, in="00:59:53:00",  out="01:00:15:01")
     
     def er_io_60(selection):
        set_selection_in_out_marks(selection, in="00:59:53:00", out="01:01:00:01")
        
      ....
```